src/test/feature/lib/file_replace.h

```diff
@@ -0,0 +1,26 @@
+#ifndef SRC_TEST_FEATURE_LIB_FILE_REPLACE_H_
+#define SRC_TEST_FEATURE_LIB_FILE_REPLACE_H_
+
+#include <string>
+#include <map>
+
+namespace hawq {
+namespace test {
+
+class FileReplace
+{
+public:
+	FileReplace() {}
+	~FileReplace() {}
```

Comment:
We don't need copy constructor here for FileReplace. We can explicitly do that in below way:
```

FileReplace() = default;
~FileReplace() = default;
FileReplace(const FileReplace&) = delete;
FileReplace& operator=(const FileReplace&) = delete;
```